float64[12] pose # 4*3 mtx
uint32   id
bool   isFixed